subhosting/manual/acceptable_use_policy.md

```diff
@@ -0,0 +1,30 @@
+# Acceptable use policy
+
+The Deno Subhosting service includes resources (CPU time, request counts) that
+are subject to this Acceptable Use policy. This document can give a rough
+estimate to what we consider as "Acceptable Use", and what we do not.
+
+### Examples of Acceptable Use
+
+- ✅ Server-side rendered websites
+- ✅ Jamstack sites and apps
+- ✅ Single page applications
+- ✅ APIs that query a DB or external API
+- ✅ A personal blog
+- ✅ A company website
+- ✅ An e-commerce site
+
+### Not Acceptable Use
+
+- ❌ Crypto mining
+- ❌ Highly CPU-intensive load (e.g. machine learning)
+- ❌ Media hosting for external sites
+- ❌ Scrapers
+- ❌ Proxy or VPN
```

Comment:
We should clarify this a little bit. Using Deploy as a reverse proxy (in front of a specific service) is allowed.